src/altinn-app-frontend/src/components/base/DatepickerComponent.tsx

```diff
@@ -279,7 +283,7 @@ function DatepickerComponent({
             maxDate={calculatedMaxDate}
             InputProps={{
               disableUnderline: true,
-              'aria-describedby': `description-${id}`,
+              'aria-describedby': textResourceBindings?.description ? `description-${id}`: undefined,
```

Comment:
this could be rewritten to `...(textResourceBindings?.description &&  {'aria-describedby':  `description-${id}`} )`